sdk/identity/identity/package.json

```diff
@@ -111,9 +111,9 @@
     "@azure/core-tracing": "^1.0.0",
     "@azure/core-util": "^1.0.0",
     "@azure/logger": "^1.0.0",
-    "@azure/msal-browser": "^2.32.2",
-    "@azure/msal-common": "^9.0.2",
-    "@azure/msal-node": "^1.14.6",
+    "@azure/msal-browser": "^2.37.0",
+    "@azure/msal-common": "^13.0.0",
```

Comment:
Please do not take direct dependency on msal-common. If there's something you need that isn't exported from msal-node or msal-browser let's chat